hack/make-rules/test-integration.sh

```diff
@@ -37,6 +37,8 @@ KUBE_TIMEOUT="-timeout 600s"
 KUBE_INTEGRATION_TEST_MAX_CONCURRENCY=${KUBE_INTEGRATION_TEST_MAX_CONCURRENCY:-"-1"}
 LOG_LEVEL=${LOG_LEVEL:-2}
 KUBE_TEST_ARGS=${KUBE_TEST_ARGS:-}
+# Default glog module settings.
+KUBE_TEST_VMODULE=${KUBE_TEST_VMODULE:-"garbagecollector*=6,graph_builder*=6"}
```

Comment:
Have you tested if the default value works as expected? I remembered the syntax was tricky.

Reply:
@ironcladlou @caesarxuchao If given`KUBE_TEST_VMODULE="''"`, the default value of  `garbagecollector=1,graph_builder=1` will not work. Is this expected?